from_mwt_ds/DataScience/notebooks/explorations.ipynb

```diff
@@ -61,7 +61,9 @@
     "people = ['Tom', 'Anna']\n",
     "topics = ['sports', 'politics']\n",
     "\n",
-    "def my_cb_simulation(n=10000, swap_after=5000, variance=0, bad_features=0):\n",
+    "def my_cb_simulation(n=10000, swap_after=5000, variance=0, bad_features=0, delta=0.8, seed=0):\n",
+    "    random.seed(seed)\n",
+    "    np.random.seed(seed)\n",
```

Comment:
I guess we don't need to print the seed here since we know what it is from the slider right?

Reply:
not sure if i understand - but we are initializing (not printing it here).